src/module/actor/character/document.ts

```diff
@@ -742,9 +742,10 @@ class CharacterPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e
             rollOptionsAll[`attribute:${key}:mod:${mod}`] = true;
         }
 
-        for (const key of SKILL_ABBREVIATIONS) {
+        for (const key of SKILL_LONG_FORMS) {
             const rank = this.system.skills[key].rank;
             rollOptionsAll[`skill:${key}:rank:${rank}`] = true;
+            rollOptionsAll[`skill:${SKILL_DICTIONARY_REVERSE[key]}:rank:${rank}`] = true;
```

Comment:
Mind commenting that this is for backward compatibility?

Reply:
Sure.